### PR TITLE
Move the command line options of phantomjs at the beggining of the exec command

### PR DESCRIPTION
**Pull request by 4lador**
The command line options were not taken in account by phantomjs because they were at the end of the command line.

Before : 
`phantomjs path/to/render.js https://www.exemple.com 1024 768 10000 PNG false --ssl-protocol=tlsv1 --ignore-ssl-errors=true`

After : 
`phantomjs --ssl-protocol=tlsv1 --ignore-ssl-errors=true path/to/render.js https://www.exemple.com 1024 768 10000 PNG false`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -160,23 +160,23 @@ Screenshot.prototype.capture = function(fn) {
     __dirname + '/script/render.js', this.url,
     this._width, this._height, this._timeout, this._format, this._clip
   ];
-  
+
+  var cliOpts = [];
```
Comment (juliangruber) [inline index.js]: this can be refactored to have `var args = []` first, then push the switches into it, then the rest of the arguments

**Comment by juliangruber** [comment]
hm i wonder why no one found this before you. maybe a prior version of phantomjs was more forgiving..

**Comment by 4lador** [comment]
Hi Julian, 

You're right, that's always better with refactoring, it is done in the last commit :) 

I don't know why no one found this before but yes that's certainly due to anything with phantomjs version, for instance I was using 1.9.0 on Linux Mint when I saw the issue.

File: index.js [hunk 2]
```diff
@@ -156,13 +156,10 @@ Screenshot.prototype.capture = function(fn) {
     }
   }
 
-  var args = [
-    __dirname + '/script/render.js', this.url,
-    this._width, this._height, this._timeout, this._format, this._clip
-  ];
-  
+  var args = []
```
Comment (juliangruber) [inline index.js]: semicolon :P then i'll merge and release

**Comment by 4lador** [comment]
Ah yes the semicolon ... my StandardJS bad habits :smiling_imp: 

That's all good now I think ^^

**Comment by juliangruber** [comment]
thanks! published as 0.6.1

**Pull request merged**